src/endpoints.js

```diff
@@ -764,6 +764,23 @@ module.exports = function (TK) {
 
   };
 
+  /**
+   * Get resources for a project
+   * @type {Function}
+   * @return {Promise}
+   */
+  TK.getProjectResources = function(data) {
+
+    var id = data.id;
+    delete data.id;
+
+    return TK.makeRequest({
+      url: '/projects/' + id + '/resources',
+      method: 'get',
```

Comment:
Remove hanging comma
```suggestion
      method: 'get'
```